### ISSUE TITLE
Uncaught TypeError: Cannot set property 'DateRange' of undefined

### ISSUE THREAD

**Issue by ksmitx** [clauderic/react-infinite-calendar#72]
Any time I try importing the calendar in a react project, I get the following error in the console:

`Uncaught TypeError: Cannot set property 'DateRange' of undefined`

It seems to be caused by an old (pre-3.0.0) version of moment-range included in the project.  The issue was fixed in newer versions of moment-range, (see: https://github.com/gf3/moment-range/issues/101) but even if I have this newer package installed in my project, the version specified in the react-infinite-calendar's package seems to be used.

Is there any way the dependency could be updated to require a newer version of moment-range to fix this issue?

**Comment by clauderic** [comment]
Hey @ksmitx,

Sorry about that. This will be addressed in the release of version 2.0, which should happen within the next couple of days.

If you'd like to test it out ahead of its release, you can run `npm install react-infinite-calendar@next` locally.

For more details, refer to #73

**Comment by ksmitx** [comment]
Great!  Good to know.  Thanks @clauderic!

**Comment by clauderic** [comment]
FYI, I just shipped version `2.0.0`, which removes the dependency on `moment` and `moment-range` in favor of `date-fns`, so will no longer be an issue moving forward.

Cheers!

**Comment by mvayngrib** [comment]
for those who want to use 1.x.x for whatever reason (in my case I'm not ready to upgrade to react 15.4), you can use webpack's imports-loader plugin to disable moment-range's AMD build:

```js
{
  test: /node_modules\/moment-range\/(dist|lib)\/moment-range(\.min)?\.js$/,
  loader: 'imports?define=>false'
}
```
